solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -0,0 +1,84 @@
+# Setup K8s Cluster and Deploy CORTX Stack
+
+   Please find below steps to setup K8s cluster and deploy CORTX Stack.
+
+## Prerequisite 
+### Minimum specification required for Cortx Stack
+   - RAM: 16GB
+   - CPU: 8Core
+   - DISK: Need 8 Disk with 10GB per disk.
+
+### If you have automated deployment for solution yaml, please make sure your vm having below drives available
+```
+      ls /dev/sd*
+```
+   - `/dev/sdb  /dev/sdc  /dev/sdd  /dev/sde  /dev/sdf  /dev/sdg  /dev/sdh  /dev/sdi  /dev/sdj`
+
+### Git package should be installed on the system. You can use below command to install it
+
+```
+      yum install git -y
+```
+### SElinux should be disabled
+   - Use below command to check status of SElinux.
+```
+      sestatus
+```
+   - If SElinux is enable then you can run below command to disabled SElinux.
+
+```
+      sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
+```
+
+   - Once above command ran successfully then reboot your system.
+
+```
+      reboot
+```   
+### Note
+ 1. All nodes should be reachable over SSH with root user
+ 2. You also deploy CORTX on AWS EC2 instance. Please follow [CORTX Deployment on AWS](https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md)
+
+## Install K8s cluster
+
+   - Clone cortx-re repository and change directory to `cortx-re/solutions/kubernetes`
+```
+      git clone https://github.com/Seagate/cortx-re 
+      cd $PWD/cortx-re/solutions/kubernetes
+```
+   - Create hosts file by below command, change root user password in below command.
+```
+      echo "hostname=$(hostname),user=root,pass=rootuserpassword" > hosts && cat hosts
+```
+   - Execute `cluster-setup.sh` to setup K8s cluster on your VM.
+
+## Deploy Cortx Stack on K8s Cluster
+
+```
+      ./cluster-setup.sh true
```

Comment:
- This command should be in the above `Install K8s cluster` section. 
- Please add that we need to add true only in case of a single node setup to untaint the primary node.